src/gwc/src/test/java/org/geoserver/gwc/layer/CatalogConfigurationTest.java

```diff
@@ -364,7 +364,7 @@ private void testModifyLayer(GeoServerTileLayer orig) {
         doThrow(new IllegalArgumentException("failedSave")).when(tileLayerCatalog).save(
                 eq(forceState2));
 
-        verify(mockMediator, never()).layerRemoved(anyString());
```

Comment:
Hmm... given the bug description wouldn't it be better to actually check the tiles are gone on disk? However that would require a non mock test. Just an idea, it's ok like this too I assume (just don't know GWC notifications well enough to say "aha, yeah, that does it")